### PR TITLE
Scotland Class 5 - Megumi Yanaka - Zoo webpage - HTML/CSS - Week 1

### PR DESCRIPTION
**Pull request by MegumiYanaka**

### CODE REVIEW

File: zoo-css-challenge/style.css [hunk 1]
```diff
@@ -105,33 +114,55 @@ footer {
...
-  background-color: #465775;
+  background-color: #483C46;
   color: #56e39f;
 }
 
 .giraffe h2 {
   color: #ef6f6c;
 }
 
+p:first-of-type{
```
Comment (bonboh) [inline zoo-css-challenge/style.css:144]:
```suggestion
.giraffe p:first-of-type{
```
The task is to make the first paragraph larger in the giraffe section, so we can add `.giraffe` to make sure we only increase the size in the giraffe section

File: zoo-css-challenge/style.css [hunk 2]
```diff
@@ -187,9 +229,13 @@ footer {
 .membership strong {
   color: #ff7f11;
 }
+.membership p {
+  max-width:700px;
+  margin: 30px 280px 30px 280px;
```
Comment (bonboh) [inline zoo-css-challenge/style.css:234]:
```suggestion
  margin: auto;
```
Alternatively, we can use `margin: auto` to center the element. `auto` evenly distributes the margin

**Review by bonboh — APPROVED** [review]
Great job @MegumiYanaka! I've left a few suggestions and tips but there's nothing else you need to do

File: zoo-css-challenge/style.css (hunk 1 above)
Comment (MegumiYanaka) [reply zoo-css-challenge/style.css:144]: Thank you for your advice!

File: zoo-css-challenge/style.css (hunk 2 above)
Comment (MegumiYanaka) [reply zoo-css-challenge/style.css:234]: Thank you, if I want to make more space between <p>and<button> in <section class="membership">, do I need to add "margin-bottom:  px;" in style?

Comment (bonboh) [reply zoo-css-challenge/style.css:234]:
```suggestion
  margin: 30px auto;
```

@MegumiYanaka Ah yes, if you want to have `30px` margin top and bottom like before then you can do `margin: 30px auto`, which is the shorthand way to do `30px` vertically and `auto` horizontally. Hope that makes sense

**Pull request closed without merging**